R/uvhydrograph-data.R

```diff
@@ -20,19 +21,23 @@ parseUVData <- function(data, plotName, month) {
     meas_Q <- subsetByMonth(getFieldVisitMeasurementsQPoints(data), month)  
     
     primary_lbl <- getUvLabel(data, "primarySeries")
+    dvmin_lbl <- getUvLabel(data, "derivedSeriesMin")
```

Comment:
hmmm seems to work with the uvhydro-example.json data for me. I'll test it out after the meeting I am about to go to  (only half hour)